Makefile

```diff
@@ -8,7 +8,21 @@ ci:
 	docker-compose up -d
 
 build:
-	docker build -t datagov/catalog.data.gov:latest ckan/
+	docker build \
+		-t datagov/catalog.data.gov:latest \
+		--build-arg CKAN_URL=http://ckan:5000 \
+		ckan/
+	docker build -t datagov/catalog.data.gov.solr:latest solr/
+	docker build -t datagov/catalog.data.gov.db:latest postgresql/
+	docker-compose build
+
+build-saml2:
```

Comment:
Why can't we just use an environment variable to change the configuration at run-time. Why do we need a special build step?